src/app/main.py

```diff
@@ -20,34 +26,41 @@
 # Initialize the Streamlit session state keys
 setup.initialize()
 
-st.title("Neglected Diagnostics: Perform Genetic Testing At Scale!")
-# Streamlit form to capture search conditions
-with st.form("query"):
-    database = st.selectbox("Select the database to search", ("nucleotide", "gene"))
-    search_term = st.text_input(
-        label="Enter the search term",
-        placeholder="Example: human[organism] AND topoisomerase[protein name]",
-    )
-    submitted = st.form_submit_button("Submit")
-    if submitted:
-        # search term is empty
-        if search_term == "":
-            st.error("Please input a search term before you submit")
-        else:
-            st.session_state[NCBI_SUMMARY_FORM] = True
-            st.session_state[NCBI_DF] = ncbi.get_data(database, search_term)
-            preprocessing.format_ncbi_summary()
-
-df_aggrid = pd.DataFrame()
-if st.session_state[NCBI_SUMMARY_FORM]:
-    aggrid_table = aggrid_table()  # Initialize and Render Aggrid Table
-    df_aggrid = aggrid_table["data"]
-
-# Streamlit UI to filter the dataset
-df_filtered = filter_dataframe(df_aggrid)
-if st.session_state[NCBI_DF_FILTER]:
-    st.dataframe(df_filtered)
-    st.write(f"Size of final data: {len(df_filtered)}")
-
-    if st.button("Prepare Download"):
-        download_data(database, df_filtered["Id"].unique(), search_term)
+# Divides the page layout into two columns of relative width
+query_col, summary_col = st.columns(MAIN_PAGE_COLS_SIZE, gap=MAIN_PAGE_COLS_GAP)
+
+with query_col:
+    st.header("Neglected Diagnostics: Democratizing Genetic Testing!")
+    # Streamlit form to capture search conditions
+    with st.form("query"):
```

Comment:
Just FYI (in case it helps during your Code Review) - I have just wrapped the existing UI into separate column. Rest of the functionality remains the same.

Thank you!